streampipes-pipeline-management/src/main/java/org/apache/streampipes/manager/api/extensions/IExtensionsProvider.java

```diff
@@ -16,17 +16,13 @@
  *
  */
 
-package org.apache.streampipes.storage.api;
+package org.apache.streampipes.manager.api.extensions;
 
-import org.apache.streampipes.model.client.endpoint.ExtensionsServiceEndpoint;
+import org.apache.streampipes.model.extensions.ExtensionItemDescription;
 
 import java.util.List;
 
-public interface IExtensionsServiceEndpointStorage {
```

Comment:
This interface seems not to be used anymore, can  it be deleted?

Reply:
deleted